packages/web-components/src/components/ic-side-navigation/ic-side-navigation.tsx

```diff
@@ -330,38 +334,34 @@ export class SideNavigation {
     }
   };
 
-  private animateCollapsedIconLabels = () => {
+  private transitionEndHandler = () => {
     const primaryNavigationWrapper = this.el.shadowRoot.querySelector(
       ".primary-navigation"
     );
+
     const secondaryNavigationWrapper = this.el.shadowRoot.querySelector(
       ".bottom-wrapper > .secondary-navigation"
     );
 
-    const collapsedIconLabelsEnd = "collapsed-icon-labels-end";
-    const collapsedIconLabelsStart = "collapsed-icon-labels-start";
-
     if (primaryNavigationWrapper) {
-      primaryNavigationWrapper.classList.remove(collapsedIconLabelsEnd);
-      primaryNavigationWrapper.classList.add(collapsedIconLabelsStart);
+      primaryNavigationWrapper.classList.remove(
```

Comment:
just checking the right classes have been removed\added??? looks like they have been switched, is that intended?